reporters_db/data/reporters.json

```diff
@@ -13685,6 +13732,30 @@
             "variations": {}
         }
     ],
+    "N.B.R. (Supp.)": [
```

Comment:
I think this may be a dupe of 

`"N.B.R. (Supp.)":`

Reply:
I don't understand, which one is it a duplicate of?

Reply:
I think there already is a reporter N.b.r. sup in the system.  ?

Reply:
The only one i can find is this one: "N.B.R."

Reply:
Okay Ive refreshed myself.  It looks like NBR is in the reporters db twice.  
N.B.R. and 
Nat. Bank. Reg. 

Can you double check which one is the official citation and merge them and then match the new one to it?

Reply:
Sure, no problem